### ISSUE TITLE
[Beta] All Backend Project Tests are Broken 

### ISSUE THREAD

**Issue by vkWeb** [freeCodeCamp/freeCodeCamp#16707]
### Issue Description
All backend project tests eg. [Metricimperial Converter](https://beta.freecodecamp.org/en/challenges/information-security-and-quality-assurance-projects/metricimperial-converter) where we need to put our project's link seems to be broken. On entering `http://example.com/` all test gets passed and I'm pushed to the next challenge. 

#### Challenges which needs to be fixed: 
- [ ] All API and Microservice Projects.
- [ ] All Information Security and Quality Assurance Projects.
 
### Browser Information
* Browser Name, Version: Chrome & Firefox(latest).
* Operating System: Windows 10 Pro 32 Bit. 
* Mobile, Desktop, or Tablet: Desktop.

**Comment by jkao1** [comment]
This is an issue with our Glitch request. We are encountering a `Unexpected token < in JSON at position 0` error because Glitch is sending back HTML. The error contained within the HTML:

```html
<noscript>
  <h1>
    To use Glitch, please enable JavaScript
  </h1>
</noscript>
```

The issue is already being addressed in #16660.

**Comment by raisedadead** [comment]
Thanks @jkao1
